src/app-layer-ssh.c

```diff
@@ -914,10 +932,18 @@ static int SSHParserTest15(void)
     Packet *p = NULL;
 
     uint8_t sshbuf1[] = "SSH-2.0-MySSHClient-0.5.1\r\n";
+    // clang-format off
```

Comment:
can the script be updated to recognize the pattern below? 4 arrays, can just use a single off/on

Reply:
Its becoming a gnarly regex.  Probably easier to not apply it to signal line arrays.